sinks/lightstep/lightstep.go

```diff
@@ -84,14 +84,19 @@ func NewLightStepSpanSink(collector string, reconnectPeriod string, maximumSpans
 
 	tracers := make([]opentracing.Tracer, 0, lightstepMultiplexTracerNum)
 
+	plaintext := true
```

Comment:
Hm, I kinda feel like we should default to encrypted and make plaintext the exception... Not sure if there are other protocol schemes out there, but seems like a good thing to encourage anyway (:

Reply:
Adjusted!